browser/src/control/jsdialog/Widget.FormulabarEdit.js

```diff
@@ -259,6 +245,14 @@ function _formulabarEditControl(parentContainer, data, builder) {
 			L.DomUtil.addClass(cursor, 'hidden');
 	});
 
+	var blockOption = JSDialog._scrollIntoViewBlockOption('nearest');
+	builder.map.on('messagesdone', function() {
```

Comment:
this should be deregistered at some point.... easier to have it as setTimeout as it was